### PR TITLE
bring backfill page back and add banner

### PR DESCRIPTION
**Pull request by jamiedemaria**
## Summary & Motivation
Keeps the backfill page for everyone regardless of what version of the runs feed they are using. We have a banner explaining that the backfills will be moving and linking to the Runs page 

Screenshot 2024-10-31 at 9 45 54 AM

The experience is a bit odd if they have the legacy runs page enabled because the link to the runs page does not include backfills. I'm not quite sure what to do about this, but want to start the review process while we figure it out

An alternative is to still hide the backfill page if the new runs feed is being used, and add a banner to the Overview page that says "backfills have moved to the Runs page, or you can revert to the old behavior"

## How I Tested These Changes
👀

### CODE REVIEW

**Comment by jamiedemaria** [comment]
* **#25663** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25663?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jamiedemaria and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx [hunk 1]
```diff
@@ -103,6 +106,41 @@ export const InstanceBackfills = () => {
...
+
+    return (
+      <Box padding={8} border="top-and-bottom">
+        <Alert
+          title={
+            <>
+              <span>Backfills are moving:</span>
+              <span style={{fontWeight: 'normal'}}>
+                {' '}
+                We&apos;re incorporating backfills into the <Link to="/runs/">Runs</Link> page to
```
Comment (jamiedemaria) [inline js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx:127]: could use some wordsmithing - working on that now

Comment (jamiedemaria) [reply js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx:127]: We're incorporating backfills into the Runs page to simplify the UI and provide one page to see what is running. 

We're incorporating backfills into the Runs page to simplify the UI. The Backfills page will be removed in a future release.

**Review by salazarm — APPROVED** [review]

File: js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx (hunk 1 above)
Comment (salazarm) [reply js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx:127]: Maybe "to see **everything that** is running. "

Comment (salazarm) [reply js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx:127]: >simplify the UI
unify the UI?

Comment (OwenKephart) [reply js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx:127]: a slight edit option: "We're incorporating backfills into the Runs page to provide a single page to see all of your executions. The Backfills page will be removed in a future release."

reason being that no all things we show on these pages are in progress (i.e. running)

**Pull request merged**